docs/operators/transform.md

```diff
@@ -0,0 +1,137 @@
+
+The operator class is TransformOperator located in the package com.datatorrent.lib.transform.
+
+Use Case
+--------
+
+Consider the data received from input source and want to transform the data as below output schema while emitting to downstream operator. 
+
+Consider input objects with these fields:
+
+`{FirstName, LastName, Phone, DateOfBirth, Address}`
+ 
+ and output objects with fields: 
+ 
+ `{Name, Phone, Age, Address}`. 
+ 
+Suppose Name is a concatenation of FirstName and LastName and 
+        Age is computed by subtracting the DateOfBirth from the current year.
+
+These simple computations can be expressed as Java expressions where the input object is
+represented by $ and provided as configuration parameters as follows:
+
+```
+Name => {$.FirstName}.concat(\" \").concat({$.LastName})
+Age => (new java.util.Date()).getYear() - {$.dateOfBirth}.getYear()
+```
+
+#### Configuration Parameters
+-----------------------------
+
+-   ***expressionMap*** -   Map<String, String>
+    -   Mandatory Parameter
+    -   Specifies the map between the output field and expression using fields of input Java object.
+
+-   ***expressionFunctions*** -   List<String>
+    -   List of imported classes or methods should be made available to expression to use. It overrides the default list.
+        Consider the same example which is explained under use case section and suppose DateOfBirth field is of type org.joda.time.DateTime.
+        org.joda.time.DateTime class is not present in the default list. So, we need to add this in expressionFunctions as below:
+        ```
+        TransformOperator operator = new TransformOperator();
+        operator.setExpressionFunctions(Arrays.asList("org.joda.time.DateTime", org.apache.commons.lang3.StringUtils.*));
+        ```
+        
+    -   Default Value = {java.lang.Math.*, org.apache.commons.lang3.StringUtils.*, org.apache.commons.lang3.StringEscapeUtils.*, org.apache.commons.lang3.time.DurationFormatUtils.*, org.apache.commons.lang3.time.DateFormatUtils.*}
+
+-   ***copyMatchingFields*** -   boolean
+    -   Specifies whether the name and type of the same fields between input and output should be copied as is or not.
+    -   Default Value = true.
+    
+#### Ports
+----------
+
+-   ***input*** -   Port for input tuples.
+    -   Mandatory input port
+    
+-   ***output***    -   Port for transformed output tuples.
+    -   Mandatory output port
+
+#### Attributes
+---------------
+-   ***Input port Attribute - input.TUPLE\_CLASS*** - Fully qualified class name.
+    -   Mandatory attribute
+    -   Type of input tuple.
+
+-   ***Output port Attribute - output.TUPLE\_CLASS*** - Fully qualified class name.
+    -   Mandatory attribute
+    -   Type of output tuple.
+    
+#### Application Example
+------------------------
+
+Please refer [Example] (https://github.com/DataTorrent/examples/tree/master/tutorials/transform) for transform sample application.
```

Comment:
This link does not render properly because of the extra space. Please review the page with mkdocs to ensure that it is rendered properly.